CONTRIBUTING.md

```diff
@@ -1,18 +1,15 @@
-## Contributing to CircuitVerse
+# Contributing to CircuitVerse
+[Back to `README.md`](README.md)
 
-- Thanks for checking out our guide. We'll be glad to hear from you.
-- Interested in making a contribution? Read on!
+Thanks for checking out our contribution guide, and we appreciate any help that you give towards CircuitVerse, no matter how small.
```

Comment:
Can you suggest some other way of saying it. Any help that you give sounds strange